### ISSUE TITLE
small contribution: sc-im interface for changing tmsu tags

### ISSUE THREAD

**Issue by nkh** [oniony/TMSU#258]
A draft of an add on bash script, I plan to add to my file manager,  which uses sc-im as an interface to tmsu. It needs to be beefed up  a bit but it's a start.

![tmsu_scim](https://user-images.githubusercontent.com/45409/197774120-ff4fbc3c-f6b2-47c5-aa69-7f3baa87d4fc.png)

https://gist.github.com/nkh/df503e5142d66b2bf4a1ce68e0f90496

**Comment by oniony** [comment]
Just fyi, I'm planning on (hoping to) create a libtmsu and separate CLI in
the Rust rewrite. This will make integration with other apps a lot easier
as you can just load the lib.

**Comment by nkh** [comment]
No, I'm a hard core linux person, I use whole applications and pipe stuff ;)

Sure it's nice to have libs too.
